network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -118,13 +120,12 @@ class BuyersGuideProductCategory(TranslatableMixin, models.Model):
     ]
 
     @property
-    def published_product_page_count(self):
-        return ProductPage.objects.filter(product_categories__category=self).live().count()
+    def alias_of(self):
+        return BuyersGuideProductCategory.objects.filter(translation_key=self.translation_key).order_by('locale_id').first()
```

Comment:
may be something that wagtail-localizse needs to shim?

Reply:
it would also be nice if this could be a mixin, but I'm not sure how we'd do that given that it needs to filter "its own class".